app/assets/javascripts/application.js

```diff
@@ -75,5 +75,15 @@ angular.module("OpenHq", ['ngRoute', 'ngAnimate', 'ngSanitize', 'rails', 'ngFile
 
   // Don’t convert attributes to camel case
   railsSerializerProvider.underscore(angular.identity).camelize(angular.identity);
+})
 
+.run(function($rootScope, $mdDialog) {
+  $rootScope.confirm = function(title, content, callback) {
+    var confirm = $mdDialog.confirm()
+          .title(title)
+          .content(content)
+          .ok('Yes, do it!')
+          .cancel('Cancel');
+    $mdDialog.show(confirm).then(callback);
```

Comment:
Also here rather than calling a callback maybe just `return $mdDialog.show(confirm)` then you can chain the promise .then to whoever calls it.